dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -945,10 +945,16 @@ def sample(self, model: dwave.optimization.Model,
 
         Returns:
             :class:`~concurrent.futures.Future` [SampleResult]:
-                Named tuple containing nonlinear model and timing info, in a Future.
+                Named tuple containing nonlinear model and general result info
+                (such as timing information and problem data id), in a Future.
```

Comment:
```suggestion
                Named tuple containing, in a Future, the nonlinear model and general
                result information such as timing and the identity of the problem data.
```

Reply:
Named tuple is in the Future, and model and info are in the tuple.